flake.nix

```diff
@@ -21,10 +21,14 @@
     pyk.inputs.flake-utils.follows = "k-framework/flake-utils";
     pyk.inputs.nixpkgs.follows = "k-framework/nixpkgs";
     foundry.url = "github:shazow/foundry.nix/monthly"; # Use monthly branch for permanent releases
+    solc = {
+      url = "github:goodlyrottenapple/solc.nix";
```

Comment:
I've made a clone of this repo until this/something similar is merged: https://github.com/hellwolf/solc.nix/pull/1